app/src/ActionModal/components/SendModal.vue

```diff
@@ -42,6 +46,18 @@
         type="custom"
         msg="doesn't have a format known by Lunie"
       />
+      <span v-if="$asyncComputed.transactionData.updating" class="memo-span">
+        Getting address for Starname...
+      </span>
+      <TmFormMsg
+        v-else-if="$asyncComputed.transactionData.error"
+        msg="could not be resolved to an address"
+        name="Starname"
+        type="custom"
+      />
+      <span v-else class="memo-span">
+        Lunie supports <a href="https://starname.me/">IOV Starnames</a>
```

Comment:
```suggestion
        Lunie supports <a href="https://starname.me/" target="_blank">IOV Starnames</a>
```

Reply:
i don't think we need this - the placeholder is good enough

Reply:
Nobody knows what a starname is...

Reply:
I suggest adding `target="_blank"` because it always annoys me that I need to go back to the page. Specially when it is a site where I'm doing important stuff (moving my funds)